plugins/defaultplugins/ifplugin/bfd_config.go

```diff
@@ -264,14 +267,14 @@ func (plugin *BFDConfigurator) DeleteBfdAuthKey(bfdInput *bfd.SingleHopBFD_Key)
 		for _, bfds := range sessionList {
 			sourceAddr := net.IP(bfds.LocalAddr[0:4]).String()
 			destAddr := net.IP(bfds.PeerAddr[0:4]).String()
-			err := vppcalls.DeleteBfdUDPSession(bfds.SwIfIndex, sourceAddr, destAddr, plugin.vppChannel)
+			err := vppcalls.DeleteBfdUDPSession(bfds.SwIfIndex, sourceAddr, destAddr, plugin.vppChannel, nil)
```

Comment:
why is last arg nil

Reply:
parent method is not called during resync